Source/SelfService/Web/microservice/microserviceView/configuration/emptyDataTable.tsx

```diff
@@ -3,27 +3,28 @@
 
 import React from 'react';
 
-import { Paper, Typography } from '@mui/material';
+import { Paper, SxProps, Typography } from '@mui/material';
 import { AddCircle } from '@mui/icons-material';
 
 import { Button } from '@dolittle/design-system/atoms/Button';
 
-type NoConfigFilesProps = {
+type EmptyDataTableProps = {
+    title: string;
+    description: string;
+    buttonText: string;
+    sx?: SxProps
     handleOnClick: () => void;
 };
 
-export const NoConfigFiles = ({ handleOnClick }: NoConfigFilesProps) => {
+export const EmptyDataTable = ({ title, description, buttonText, sx, handleOnClick }: EmptyDataTableProps) => {
```

Comment:
I like this renaming. I think I see that this "empty-state" component can be a valuable component to use in a DataGrid molecule-component exposed by the design system. But I thin that may require some more thinking about